.eslintrc.json

```diff
@@ -0,0 +1,21 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true
+  },
+  "extends": ["airbnb-base"],
+  "globals": {
+    "Atomics": "readonly",
+    "SharedArrayBuffer": "readonly"
+  },
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "rules": {
+    "arrow-parens": "off",
+    "no-unused-vars": "off",
+    "no-unused-expressions" : "off",
+    "no-nested-ternary" : "off"
+  }
```

Comment:
Please use the provided [.eslintrc.json](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json) and do not modify it.